package.json

```diff
@@ -64,5 +64,8 @@
 	"simple-git-hooks": {
 		"pre-commit": "npm run lint:type && npm run git:hook:precommit",
 		"commit-msg": "npm run git:hook:commit"
+	},
+	"dependencies": {
+		"clsx": "2.1.1"
```

Comment:
This library must be a part of the frontend workspace we do not need it for the whole project. I mean the package.json inside the frontend. Uninstall it and install again as `npm install clsx -w frontend`, and it will appear in the frontend workspace